### ISSUE TITLE
Comprehensive molecular characterization of human colon and rectal cancer

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#26]
## TL;DR

To characterize somatic alterations in colorectal carcinoma, we conducted a genome-scale analysis of 276 samples, analysing exome sequence, DNA copy number, promoter methylation and messenger RNA and microRNA expression. A subset of these samples (97) underwent low-depth-of-coverage whole-genome sequencing. Colon and rectum cancers were found to have considerably similar patterns of genomic alteration. Twenty-four genes were significantly mutated, and in addition to the expected APC, TP53, SMAD4, PIK3CA and KRAS mutations, we found frequent mutations in ARID1A, SOX9 and FAM123B. 

### Paper Link

https://www.nature.com/articles/nature11252

### Author/Institution

The Cancer Genome Atlas Network

## Methods

colorectal cancer. survival analysis. network analysis.

## Cite

The Cancer Genome Atlas Network. Comprehensive molecular characterization of human colon and rectal cancer. Nature 487, 330–337 (2012). https://doi.org/10.1038/nature11252
